modules/blockbuilder/blockbuilder.go

```diff
@@ -201,6 +201,7 @@ func (b *BlockBuilder) running(ctx context.Context) error {
 		waitTime, err := b.consume(ctx)
 		if err != nil {
 			level.Error(b.logger).Log("msg", "consumeCycle failed", "err", err)
+			waitTime = b.cfg.ConsumeCycleDuration
```

Comment:
Shouldn't `consume` decide how long to wait for? We may want to retry immediately.

Reply:
Well in this case I think that it makes sense to let the runner to decide what to do if there are no partitions

Reply:
But `consume` can fail for other reasons (eg. failed to commit). In such case, we probably want to retry immediately, not wait a full cycle.

Reply:
I see your point, I will change it